recipe/meta.yaml

```
@@ -30,9 +30,14 @@ source:
   sha256: dd422d4cc8462d112411659a56cc78cb6010d704f83db3e48c55a3f66eed5ec2  # [win]
 
 build:
-  number: 0
+  number: 1
   skip: true  # [osx]
 
+test:
+  commands:
+    - test -f $PREFIX/lib/cmake                      # [linux]
```

Somehow I slipped a typo in here. 🙃
```suggestion
    - test -d $PREFIX/lib/cmake                      # [linux]
```